extract_jira_key.py

```diff
@@ -0,0 +1,27 @@
+import sys
+import re
+
+def extract_jira_issue_key(text):
+    """
+    Extracts a Jira issue key from the input text following the /jira-epic pattern.
+    """
+    # regular expression to match the /jira-epic command and extract the issuekey
+    pattern = r"/jira-epic (\b[A-Z]+-\d+\b)"
```

Comment:
`/jira-epic\s+(\b[A-Z]+-\d+\b)`

Using `\s+` instead of just one space could be less error-prone.

Also, I wonder if we should allow a lowercase project key to make it even more friction-less.